doc/classes/Transform3D.xml

```diff
@@ -181,8 +181,25 @@
 	</members>
 	<constants>
 		<constant name="IDENTITY" value="Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0)">
-			A transform with no translation, no rotation, and its scale being [code]1[/code]. Its [member basis] is equal to [constant Basis.IDENTITY].
-			When multiplied by another [Variant] such as [AABB] or another [Transform3D], no transformation occurs.
+			The identity [Transform3D]. This is a transform with no translation, no rotation, and a scale of [constant Vector3.ONE]. Its [member basis] is equal to [constant Basis.IDENTITY]. This also means that:
+			- Its [member Basis.x] points right ([constant Vector3.RIGHT]);
+			- Its [member Basis.y] points up ([constant Vector3.UP]);
+			- Its [member Basis.z] points back ([constant Vector3.BACK]).
+			[codeblock]
+			var transform = Transform3D.IDENTITY
+			var basis = transform.basis
+			print("| X | Y | Z | Origin")
+			print("| %s | %s | %s | %s" % [basis.x.x, basis.y.x, basis.z.x, transform.origin.x])
+			print("| %s | %s | %s | %s" % [basis.x.y, basis.y.y, basis.z.y, transform.origin.y])
+			print("| %s | %s | %s | %s" % [basis.x.z, basis.y.z, basis.z.z, transform.origin.z])
+			# Prints:
+			# | X | Y | Z | Origin
+			# | 1 | 0 | 0 | 0
+			# | 0 | 1 | 0 | 0
+			# | 0 | 0 | 1 | 0
```

Comment:
As of https://github.com/godotengine/godot/pull/47502 decimals are always included when printing floats:

```suggestion
			print("| X   | Y   | Z   | Origin")
			print("| %s | %s | %s | %s" % [basis.x.x, basis.y.x, basis.z.x, transform.origin.x])
			print("| %s | %s | %s | %s" % [basis.x.y, basis.y.y, basis.z.y, transform.origin.y])
			print("| %s | %s | %s | %s" % [basis.x.z, basis.y.z, basis.z.z, transform.origin.z])
			# Prints:
			# | X   | Y   | Z   | Origin
			# | 1.0 | 0.0 | 0.0 | 0.0
			# | 0.0 | 1.0 | 0.0 | 0.0
			# | 0.0 | 0.0 | 1.0 | 0.0
```

Alternatively we can change the format specifiers to `%.f` to remove the trailing `.0`.

Reply:
Way ahead of you. I'm actually handling all of these occurrences separately in https://github.com/godotengine/godot/pull/99081 . The original PR is a bit prickly as you know, and whether it's worth doing is still up in the air.

The `%.f` is a good idea though(?)

Reply:
Updated to use the above format specifier